package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "dev": "next dev",
+    "dev": "next dev --turbo",
```

Comment:
Maybe create a new script called turbo that is `next dev --turbo` instead of using the main dev one
The reason is that turbo is technically not stable yet (it is in next 15 i think)

Reply:
Didn't see that I included this lol. Was just playing around with turbopack. I'll make the alternate script